setup.py

```diff
@@ -6,14 +6,15 @@
   packages  = ['rest'],
 
   install_requires = [
-    'Flask         == 0.10',
-    'unicodecsv    == 0.9.4',
+    'Flask         == 1.0.4',
```

Comment:
we're using flak 1.0.1 internally, should we use that here too?

Reply:
ah yeah i'll update this, should be fine